### DISCUSSION TITLE
AppleTv quirks without original Apple remote

### DISCUSSION THREAD

**Discussion by iRonin** [jellyfin/Swiftfin#1291] [Discussions]
My AppleTv remote broke long time ago and I've been using either the TV remote or my iPhone to control it which usually works but Swiftfin has some quirks:

- I cannot get the subtitles menu with the iPhone remote, pressing down works fine with my TV remote
- I cannot pause/resume playback with my TV remote, for this I have to use the dedicated play/pause button on iPhone remote (usually playback apps start stop when pressing the touchpad; I think I it works only when I go to playback slider where I can slide to point in time of the playback)

It seems that without the original AppleTV remote I have to use two, or am I missing something? 

Thanks!

**Comment by LePips** [comment]
Considering a duplicate of a few other issues already reported, like #914. You can search other issues regarding the touchpad remote options (physical and iOS).
